utils/logalloc.cc

```diff
@@ -1271,14 +1263,26 @@ reclaim_timer::reclaim_timer(const char* name, is_preemptible preemptible, size_
     , _segments_to_release(segments_to_release)
     , _tracker(tracker)
     , _debug_enabled(timing_logger.is_enabled(logging::log_level::debug))
+    , _log_region_occupancy(_debug_enabled && _tracker)
```

Comment:
What's the problem the refactoring is trying to solve?

Reply:
not sure how this commit made it out of my work branch, sorry about that.